src/app/FakeLib/CscHelper.fs

```diff
@@ -78,11 +78,22 @@ let cscExe toolPath (srcFiles : string list) (opts : string list) : int =
 ///                                Target = ...
 ///                                ... })
 let csc (setParams : CscParams -> CscParams) (inputFiles : string list) : int =
-    let inputFiles = inputFiles |> Seq.toList
+    // Helper to quote a path with spaces in it, if not already quoted.  See https://github.com/fsharp/FAKE/issues/992
+    let ensureTrimQuotedPath (path : string) =
```

Comment:
Actually, I'm hesitant about making this a general function.  Digging around, this is [not as simple as it first appears](http://blogs.msdn.com/b/twistylittlepassagesallalike/archive/2011/04/23/everyone-quotes-arguments-the-wrong-way.aspx).  Might be better to leave it local to the `csc` function.  It is already local in the other places.